core/templates/dev/head/pages/landing/fractions/Fractions.js

```diff
@@ -22,16 +22,24 @@ oppia.controller('Fractions', [
   function($scope, $timeout, $window,
       siteAnalyticsService, UrlInterpolationService) {
     $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+
     $scope.getStaticSubjectImageUrl = function(subjectName) {
       return UrlInterpolationService.getStaticImageUrl('/subjects/' +
         subjectName + '.svg');
     };
+
     $scope.onClickGetStartedButton = function(viewerType) {
       siteAnalyticsService.registerOpenFractionsFromLandingPageEvent(
         viewerType);
       $timeout(function() {
         $window.location = ' /collection/4UgTQUc1tala';
```

Comment:
Why is there a leading space in the string?

Reply:
Oops fixed that.